plugins/cros-ec/fu-cros-ec-usb-device.c

```diff
@@ -271,6 +284,8 @@ fu_cros_ec_usb_device_setup (FuDevice *device, GError **error)
 		return FALSE;
 	}
 
+	self->writeable_offset =
+		GUINT32_FROM_BE (start_resp.rpdu.common.offset);
```

Comment:
this can be allone line if you want, I'm not a nazi with line width.

Reply:
Done.